src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +119,214 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// </summary>
 		public static readonly BindableProperty ItemTemplateProperty =
 			BindableProperty.CreateAttached(nameof(ItemTemplate), typeof(DataTemplate), typeof(Shell), null, BindingMode.OneTime);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetItemTemplate']/Docs/*" />
+		/// <summary>
+		/// Gets DataTemplate applied to each ShellItem object managed by Shell.
+		/// </summary>
+		/// <param name="obj">The BindableObject that sets the DataTemplate applied to Item objects.</param>
+		/// <returns>The DataTemplate applied to Item objects.</returns>
 		public static DataTemplate GetItemTemplate(BindableObject obj) => (DataTemplate)obj.GetValue(ItemTemplateProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetItemTemplate']/Docs/*" />
+
+		/// <summary>
+		/// Sets DataTemplate applied to each ShellItem object managed by Shell.
+		/// </summary>
+		/// <param name="obj">The BindableObject that sets the DataTemplate applied to Item objects.</param>
+		/// <param name="itemTemplate">The DataTemplate applied to Item objects.</param>
 		public static void SetItemTemplate(BindableObject obj, DataTemplate itemTemplate) => obj.SetValue(ItemTemplateProperty, itemTemplate);
 
 		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetBackButtonBehavior']/Docs/*" />
 		public static BackButtonBehavior GetBackButtonBehavior(BindableObject obj) => (BackButtonBehavior)obj.GetValue(BackButtonBehaviorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetBackButtonBehavior']/Docs/*" />
+
+		/// <summary>
+		/// Sets the behavior of the back button.
+		/// </summary>
+		/// <param name="obj">The BindableObject that sets the behavior of the back button.</param>
+		/// <param name="behavior">Sets the back button behavior.</param>
 		public static void SetBackButtonBehavior(BindableObject obj, BackButtonBehavior behavior) => obj.SetValue(BackButtonBehaviorProperty, behavior);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetPresentationMode']/Docs/*" />
+		/// <summary>
+		/// Gets the navigation animation that occurs when a page is navigated to with the GoToAsync method.
+		/// </summary>
+		/// <param name="obj">The BindableObject that modify the Shell tabs visibility.</param>
+		/// <returns>The navigation animation that occurs when a page is navigated.</returns>
 		public static PresentationMode GetPresentationMode(BindableObject obj) => (PresentationMode)obj.GetValue(PresentationModeProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetPresentationMode']/Docs/*" />
+
+		/// <summary>
+		/// Sets the navigation animation that occurs when a page is navigated to with the GoToAsync method.
```

Comment:
This is a long summary. Have you seen how it displays on intellisense? I love that you're explicitly definig what the different presentation modes do, but I'm not sure this is the right formatting for a multiline comment. 

I know that [**Elements.cs**](https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Element/Element.cs) did something similar, but the formatting here differs.